assets/airports/airportLoadList.js

```diff
@@ -72,6 +72,12 @@ window.AIRPORT_LOAD_LIST = (function() { // eslint-disable-line wrap-iife
             wip: true
         },
         {
+            icao: 'efhk',
+            level: 'easy',
```

Comment:
I don't think easy is appropriate for this airport, maybe medium?

Reply:
I would say keep this at medium.

Reply:
Since then there were guidelines introduced which specify which difficulty rating and airport should have read here: https://github.com/openscope/openscope/blob/develop/assets/airports/airportLoadList.js#L34